java-container/google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java

```diff
@@ -26,7 +26,6 @@
 import com.google.container.v1.ListOperationsResponse;
 import com.google.container.v1.NodePool;
 import com.google.container.v1.Operation;
-import com.google.container.v1.Operation.Status;
```

Comment:
Manual commit to monorepo. OK to have diff https://github.com/googleapis/google-cloud-java/commit/8e4378ddb38c2ff4d50b65962ce729c2b237c095